### PR TITLE
[FEAT-PAS-4] Added base component, module and electron support

### PR DESCRIPTION
**Pull request by gabrielpda15**
# Description

Added base component, module and electron support

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update
-   [x] This change includes new libraries
-   [ ] This change includes new migrations

# How has this been tested?

There is nothing to test.

**Test Configuration**:

-   OS: Windows
-   Var Envs: None

**Test Workflow:**

There is nothing to test.

# Checklist:

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] New and existing unit tests pass locally with my changes
-   [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,5 +21,27 @@
...
+		"copyfiles": "^2.4.1",
+		"electron": "^22.0.3",
+		"nodemon": "^2.0.20",
+		"rimraf": "^4.1.1",
+		"sass": "^1.57.1",
+		"typescript": "^4.9.4"
+	},
+	"peerDependencies": {
+		"@popperjs/core": "2.11.6"
+	}
```
Comment (gabrielpda15) [inline package.json:46]: Essa peer dependency é obrigatória? [exemplo de comentario]

File: src/main.ts [hunk 2]
```diff
@@ -1 +1,61 @@
-console.log('Hello Feeders!');
+import 'reflect-metadata';
+import { ElectronReloader } from './reloader';
+import { resolve as pathResolve } from 'path';
+import { app as electronApp, BrowserWindow } from 'electron';
+import { config } from './config';
+
```
Comment (gabrielpda15) [inline src/main.ts:6]: Volta com o hello feeders! Gosto dele! [exemplo de RC]
```suggestion
import 'reflect-metadata';
import { ElectronReloader } from './reloader';
import { resolve as pathResolve } from 'path';
import { app as electronApp, BrowserWindow } from 'electron';
import { config } from './config';

console.log('Hello Feeders!');
```

**Review by gabrielpda15 — COMMENTED** [review]
Muito bonito! Mas pode melhorar!

File: package.json (hunk 1 above)
Comment (gabrielpda15) [reply package.json:46]: Sim bobão! [exemplo resposta]

File: src/main.ts (hunk 2 above)
Comment (gabrielpda15) [reply src/main.ts:6]: Não! [exemplo resposta]

**Review by Utahh — APPROVED** [review]
Revised and ready for the next one.

**Review by Utahh — APPROVED** [review]
Revised and ready for the next one.

**Review by ademirj93 — APPROVED** [review]
i don`t intendi much things, but i confio in you

**Pull request merged**